cli/azd/resources/apphost/templates/containerApp.tmpl.yamlt

```diff
@@ -35,7 +35,7 @@ properties:
 {{- if .Ingress}}
     ingress:
       external: {{ .Ingress.External }}
-      targetPort: {{ .Ingress.TargetPort }}
+      targetPort: {{ "{{ .TargetPort }}" }}
```

Comment:
Remember to remove the `TargetPort` from the `generate.go` files, as we won't be using that one any more